py/pykms/pykmsbase.cpp

```diff
@@ -77,10 +77,10 @@ void init_pykmsbase(py::module &m)
 			.def("__repr__", [](const Plane& o) { return "<pykms.Plane " + to_string(o.id()) + ">"; })
 			;
 
-	py::enum_<PlaneType>(m, "PlaneType")
-			.value("Overlay", PlaneType::Overlay)
-			.value("Primary", PlaneType::Primary)
-			.value("Cursor", PlaneType::Cursor)
+	py::enum_<Plane::Type>(m, "Type")
```

Comment:
Yes... I though it would be pykms.Plane.Type.Overlay, but it really is pykms.Type.Overlay. I change that to PlaneType (this is also good because now all py-scripts that had plane types in them do not need any changes).